### PR TITLE
Updating NET sdk requirement

### PR DESCRIPTION
**Pull request by manishtr**
### Change Description ###

Updating the .NET Core SDK requirement

### CODE REVIEW

File: docs/application/vscode-ext/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Visual Studio Code Extension for Tizen enables you to develop Tizen .NET and Tiz
 
   To use VS Code Extension for Tizen, you must install the following tools:
 
-  - NET Core SDK 2.0 or later **for Tizen.Net only**
+  - NET Core SDK 2.0 **for Tizen.Net only**
```
Comment (journey2w) [inline docs/application/vscode-ext/index.md]:
```suggestion
  - NET Core SDK 2.0 **for Tizen .Net only**
```

Please see the Tizen naming rules: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md.

**Comment by journey2w** [comment]
@manishtr
Please fill in the PR template.
It would be great if you write the reason why the requirement is changed.

I can guess the reason, but other can't know it. (Perhaps, you won't maintain VS Code Extension anymore.)

**Comment by mijin85cho** [comment]
@manishtr Would you please check the review comment and reply to it?
We are waiting your response to proceed this PR.

**Comment by mijin85cho** [comment]
I checked with @surbhig12 to close this PR.

"The changes need to be confirmed. Therefore, closing the PR for now and will reopen later."

**Pull request closed without merging**